lib/dns/server.js

```diff
@@ -535,6 +536,16 @@ class RecursiveServer extends DNSServer {
       this.stubPort = options.stubPort;
     }
 
+    if (options.noUnbound === true) {
```

Comment:
I know we discussed this out of band, but now I think we should fully validate `noUnbound` here, i.e.:
```js
if (options.noUnbound != null) {
  assert(typeof options.noUnbound === 'boolean')
  if (options.noUnbound) {
    ...
  }
}
```

The motivation is to protect against one-off imports of this module that do not use `bcfg` to validate certain properties of the options hash.

Reply:
Good call. Pushed up a fix for this.